settings.py

```diff
@@ -56,7 +59,7 @@ class MainSettings(BaseSettings):
     PORT: int = Field(default=8000)
     WORKERS_COUNT: int = Field(default=1)
     TRUSTED_HOSTS: List[str] = Field(default=["*"])
-    DATETIME_FORMAT: str = Field("%Y-%m-%d %H:%M:%S")
+    DATETIME_FORMAT: str = Field("%Y-%m-%d %H-%M-%S")
```

Comment:
Why did you change whole project datetime format, please refer to ISO-8601:
YYYY-MM-DDTHH:MM:SS+-HHMM

Reply:
It was not good attempt to avoid error with service file creating. Of course I'll fix it now